### DISCUSSION TITLE
Always on controls for default layout

### DISCUSSION THREAD

**Discussion by sukantatoptal** [vidstack/player#1294] [Q&A]
I am using vid stack player with default layout. Also same video view  for both audio and video files. I have a requirement to always show controls but i am not able to figure out if its possible. 

After trying to set the control delay to a large number, it still disappears when I click on the video in mobile view. Does anyone know how to accomplish this?

**Comment by Curetix** [comment]
You can pause the controls idle tracking.

```ts
player.controls.pause();
// or
const remote = useMediaRemote();
remote.pauseControls();
```

Depends a bit on your setup, are you using React or Web Components?
